file-formats/susi/examples/zaff_example_susi.susi.json

```diff
@@ -0,0 +1,24 @@
+{
+    "formatVersion": "1",
+    "header": {
+        "description": "Example SUSI object",
+        "originalLanguage": "en",
+        "abapLanguageVersion": "standard"
+    },
+    "general": {
+        "applName": "Z_FEM3/Z_FEM3_GET_INFO",
+        "type": "WebService",
+        "hashName": "D2FF3796C3963799AE24F0224FCD96HS",
```

Comment:
I am wondering wether the `hashName` is the actual object name, isn't it? 

If so, we should name the example object's file name to `D2FF3796C3963799AE24F0224FCD96HS.susi.json` and remove it from the file's content.

Reply:
This is my first example, I think as soon as the backend is finished, I will create new example file

Reply:
Maybe, this is a misunderstanding. I am still not sure. 

If `hashCode` was the name of the example repository object itself, it shouldn't be part of the content (we should remove the field `hashCode`), but part of the file name. 
If it was a reference to another object, it would be fine to have it as part of the file content.

Reply:
done